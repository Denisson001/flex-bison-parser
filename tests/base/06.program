{
    read(line_s);
    flag = 1;
    while (k < len(line_s)) {
        if (line_s[k] == "(") {
            sum = sum + 1;
        } else {
            sum = sum - 1;
        }
        if (sum < 0) {
            flag = 0;
        }
        k = k + 1;
    }
    if (flag == 1) {
        if (sum == 0) {
            print("True");
        } else {
            print("False");
        }
    } else {
        print("False");
    }
}