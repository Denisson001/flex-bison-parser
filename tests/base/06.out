28
16
-1
-1
